backend/pkg/rest.utils/paging.go

```diff
@@ -69,14 +72,9 @@ func ParsePagingParameters(r *http.Request) (int64, int64, error) {
 	} else {
 		perPage, err = strconv.ParseInt(qPerPage, 10, 64)
 		if err != nil {
-			return -1, -1, errors.Errorf(
-				"invalid per_page query: \"%s\"",
-				qPerPage,
-			)
-		} else if perPage < 1 {
-			return -1, -1, errors.New("invalid per_page query: " +
-				"value must be a non-zero positive integer",
-			)
+			return -1, -1, ErrQueryParmInvalid("per_page", qPerPage)
+		} else if page < 1 {
```

Comment:
This condition is using the wrong parameter.
```suggestion
		} else if perPage < 1 {
```
Why didn't the CI catch this? :thinking: